File: 2.0/java/calls/findCallsPharmacyReminders.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.callstexts.model.Call;
import com.callfire.api.client.api.callstexts.model.request.FindCallsRequest;
import com.callfire.api.client.api.common.model.Page;

import java.util.Calendar;
import java.util.GregorianCalendar;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        // find all calls made through campaign with id = 10, label = reminders is assigned to such campaign,
        // between 2015-12-01 and 2015-12-10, return first 10 records if any
        FindCallsRequest request = FindCallsRequest.create()
            .campaignId(10L)
            .label("reminders")
            .intervalBegin(new GregorianCalendar(2015, Calendar.DECEMBER, 1, 0, 0, 0).getTime())
            .intervalEnd(new GregorianCalendar(2015, Calendar.DECEMBER, 10, 0, 0, 0).getTime())
            .limit(10L)
            .build();
        Page<Call> calls = client.callsApi().find(request);
        // see Call.records.questionResponses list for stored data
    }
}

File: 2.0/csharp/calls/findCallsPharmacyReminders.cs
using System;
using CallfireApiClient;
using CallfireApiClient.Api.CallsTexts.Model;
using CallfireApiClient.Api.CallsTexts.Model.Request;
using CallfireApiClient.Api.Common.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");

        var request = new FindCallsRequest
        {
            CampaignId = 10,
            Label = "reminders",
            IntervalBegin = new DateTime(2016, 12, 1, 0, 0, 0),
            IntervalEnd = new DateTime(2016, 12, 10, 0, 0, 0),
            Limit = 10
        };
        Page<Call> calls = client.CallsApi.Find(request);
        // check Call.records.questionResponses for stored data
    }
}
